packages/core/src/components/Box.ts

```diff
@@ -51,16 +52,17 @@ export interface TuiBoxProps extends Omit<Styles, 'textWrap'> {
 }
 
 export const TuiBox: FunctionalComponent<TuiBoxProps> = (props, { slots }) => {
+  const normalizedProps = pickBy(props, (value) => value !== undefined)
```

Comment:
Let's remove this normalizedProps since it's not needed: just setting `TuiBox.props` is enough to generate the good casing

Reply:
Oh,That's really not a good solution ! I found out why I add it.Because the `display` props `undefined`  is treated as `none` to causing them to not work normal.we can just set the default value as `flex` directly.